style.css

```diff
@@ -412,12 +412,111 @@ header {
 }
 
 .skills {
-  /* margin-left: 145px;
-  margin-right: 115px; */
   padding-right: 40px;
   margin-top: 12px;
 }
 
 .frameworks {
   padding-left: 12px;
 }
+
+.project1:hover,
+.project2:hover {
+  background-color: #daa884;
+  cursor: pointer;
+}
+
+#resume:hover {
+  background-color: #daa884;
+  cursor: pointer;
+}
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+}
+
+.contact h1 {
+  font-family: 'Inter', sans-serif;
+  font-weight: bolder;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 120px 24px 105px;
+  color: #172b4d;
+  text-align: center;
+  min-height: 216px;
+  justify-content: center;
+}
+
+.form-1 {
+  display: flex;
+  flex-direction: column;
+  margin-left: 1.5rem;
+  margin-right: 1.5rem;
+}
+
+#name,
+#email,
+#textarea-1 {
+  padding: 12px;
+  border: none;
+  margin-bottom: 14px;
+  margin-top: 39px;
+}
```

Comment:
As requested in the requirements, in order to stick as much as possible to the design, I suggest you apply the following to your textarea: 
- It should have a `top-margin` and `bottom-margin` of `24px`.
-  It also should have a font family set to `Inter`.

### The template: 
![Screenshot from 2023-01-25 20-59-17](https://user-images.githubusercontent.com/93520178/214676213-17ddf945-cb90-4ba6-92d9-73d941ce225c.png)
### Your design: 
![Screenshot from 2023-01-25 20-59-35](https://user-images.githubusercontent.com/93520178/214676280-08665063-65e9-42a5-be0d-86a3931b960c.png)